test/nbrowser/gristUtils.ts

```diff
@@ -3389,9 +3389,16 @@ export async function waitForAnchor() {
   await driver.wait(async () => (await getTestState()).anchorApplied, 2000);
 }
 
-export async function getAnchor() {
+export async function getAnchor({wait = false}: {wait?: boolean;} = {}) {
```

Comment:
Thinking about this more, what if we had a new helper that sent the keyboard shortcut keys and waited for the success toast? Something like `copyAnchor`.

Then, I think it'd be ok to update the places we call `getAnchor` to also call `copyAnchor` just before, instead of sending keys directly.

Wdyt?

Reply:
Not sure I follow? How the `getAnchor` stays the same?

Reply:
Sorry, worded that poorly. What I meant was something like:

```javascript
export async function copyAnchor() {
  await driver.find('body').sendKeys(Key.chord(Key.SHIFT, await modKey(), 'a'));
  // Wait for success toast.
}

export async function getAnchor() {
  await copyAnchor();
  return (await getTestState()).clipboard || '';
}
```

I think we could inline it all in `getAnchor` too - nothing needs the separation now. The main thing would be to wait for the toast.